src/pages/ShoppingCart/utils/getCustomerServiceLink.ts

```diff
@@ -16,9 +17,14 @@ export const getWebWhatsappLink = ({
   return url;
 };
 
-export const getInvoiceText = (shoppingCartProducts: ShoppingCartProduct[]) => {
+export const getInvoiceText = (
+  shoppingCartProducts: ShoppingCartProduct[],
+  provider: string | undefined
```

Comment:
I'm not sure if this works, please test and confirm hehe
```suggestion
  provider?: string
```

Reply:
works, thanks